.bazelrc

```diff
@@ -1 +1,2 @@
-build --cxxopt=-std=c++14
\ No newline at end of file
+build --cxxopt=-std=c++14
+build --action_env=GEM_HOME
```

Comment:
Ah this is a *really* good idea!  Some suggestions to cover this more thoroughly:
--action_env=PATH --action_env=GEM_PATH --action_env=GEM_HOME

--action_env=DOTNET_CLI_TELEMETRY_OPTOUT=1
--action_env=DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1

The latter are to cover some similar awkwardness around the current state of C#

Reply:
Would you like to see the other places where those environment variables are set migrated here, or do we need them set in multiple places?

Reply:
Yea if we could remove those from our kokoro builds and docker images that would be amazing too!  I think centralizing all of this temporary ugliness here is a better state to be in, and it brings up one step closer to getting `bazel test //...` working :) (see https://github.com/protocolbuffers/protobuf/issues/10427)

Reply:
Leaving the DOTNET changes out here, can make a follow up PR to address non-Ruby changes.